### DISCUSSION TITLE
Support for il-central-1

### DISCUSSION THREAD

**Discussion by innapos** [aws-cloudformation/cfn-lint#2851] [Q&A]
Will the tool support the il-central-1 region? Is it known when?

**Comment by kddejong** [comment]
Sorry, I have been bad at keeping up on discussions.  This should have been released within a few weeks of the release of the region.  We have automation that breaks as new regions come online.  You just need to make sure you are using the appropriate versions.

https://github.com/aws-cloudformation/cfn-lint/pull/2836
